### ISSUE TITLE
Footer customization

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#108]
Sysadmins will need to modify [the default footer](https://github.com/devilry/trix2/blob/master/trix/trix_student/templates/trix_student/include/footer.django.html) to include the service's contact information, bug reporting, etc.

**Comment by Levijatan** [comment]
Added the ability to override the footer template inn a same manner as the consent template

**Comment by torgeirl** [comment]
This might be fixed with #109, but in `3.0.0b12` there is some weird extra space when adding custom links to the footer:

![Screenshot from 2022-12-06 15-30-56](https://user-images.githubusercontent.com/796053/205939745-e9f45c93-e6b4-4c40-bb83-6086efc673f5.png)


The source contains one empty div tag (`report_bug` I assume) before accessibly, but there isn't anything extra between the `wcag` and `contact_info`:
```
(...)
<div class="trix-footer">
    <div>
        
    </div>
    <div>
            <span class="fa fa-eye-slash"></span>
            <a href="/wcag">Deaktiver tilgjengelighetsfarger</a>
    </div>
    <div>
            <a href="link1">Hjelp</a>
            <a href="link2">Personvern</a>
    </div>
</div>
        </footer>
    </body>
</html>
```

**Comment by Levijatan** [comment]
Changed it to have a block called body inside a div, instead of different blocks in their own divs. Seems to have removed the extra spacing. Created a b13 prerelease.
